client/my-sites/subscribers/controller.js

```diff
@@ -23,6 +25,21 @@ const queryStringChanged = ( key ) => ( value ) => {
 	return page.show( addQueryArgs( path, { [ key ]: value } ) );
 };
 
+export async function redirectIfInsufficientPrivileges( context, next ) {
+	const { store } = context;
+	const state = store.getState();
+	const site = getSelectedSite( state );
+	const canManageOptions = canCurrentUser( state, site.ID, 'manage_options' );
```

Comment:
There's a bunch of `*_users` capabilities, should we use `list_users` here instead?

https://wordpress.org/documentation/article/roles-and-capabilities/

Reply:
Not a blocker for merging, to be clear. `manage_options` is fine for me.